core/templates/dev/head/profile/profile.html

```diff
@@ -33,8 +33,15 @@
   <div class="oppia-profile-container" ng-controller="Profile">
     <md-card class="oppia-profile-user-card">
 
+    <div ng-if=" '{{username}}' == '{{PROFILE_USERNAME}}' ">
+      <a HREF="/preferences"> <img ng-src="<[profilePictureDataUrl]>"
```

Comment:
In this case you should put the start of the `<img...>` tag on its own line, like so:

    <a ...>
      <img ...>
    </a>